docs/reference/action_list.rst

```diff
@@ -186,6 +217,21 @@ Available types and associated options
 | ``FieldDescriptionInterface::TYPE_*`` |                     | See :doc:`Field Types <field_types>`                                  |
 +---------------------------------------+---------------------+-----------------------------------------------------------------------+
 
+Setting up custom template
```

Comment:
We already talk about custom template somewhere else, and here we talk only about actions.

I would use a title like `Setting up custom action buttons`

Reply:
Done!